### PR TITLE
Bump JWT expiry to one week

### LINKED ISSUE #246: Bump JWT expiry time
JWT expiration date is calculated _before_ an event is placed onto the queue.  If an item sits on the queue for longer than the expiry time, and is processed _after_ the expiry date, then it will necessarily fail.  This can cause cascading failures.

The stopgap solution is to bump the JWT expiry time to a high limit, e.g. one day, week, or month.

### PR DESCRIPTION
**Pull request by birkland**
This virtually assures that JWT tokens won't expire before events are processed from a slow queue.  Based on Danny's recommendation for very long JWT expiry times to avoid cascading failure.

Resolves #246

### CODE REVIEW

File: codebase/config/sync/islandora.settings.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 broker_url: 'tcp://activemq:61613'
-jwt_expiry: '+2 hour'
+jwt_expiry: '+168 hour'
```
Comment (bseeger) [inline codebase/config/sync/islandora.settings.yml:2]: I think this looks fine, but I can't necessarily test it out. 
I do wonder if you want to say  `+7 days` instead?   I don't really care or have an opinion on that.

**Review by bseeger — DISMISSED** [review]
This looks good to me. I'm only able to visually look the changes over, but time will tell if that timeout works well.

**Comment by bseeger** [comment]
@birlkand, the thing that needs to change is the variable in the `.env` file.  This [PR](https://github.com/jhu-idc/idc-isle-dc/pull/182) created a new env var in the drupal environment named: `DRUPAL_JWT_EXPIRY_INTERVAL`

This is what should be set as it will override the `jwt-expiry` in the settings.local.php file: 

```
$config['islandora.settings']['jwt_expiry'] = str_replace("'", "", str_replace('"', '', getenv('DRUPAL_JWT_EXPIRY_INTERVAL') ?: '+2 hour'));
```

Format itself (of `+168 hour`) seems fine.

**Comment by birkland** [comment]
Closed in favor of https://github.com/jhu-idc/idcApp-k8s/issues/81

**Pull request closed without merging**